frigate/events.py

```diff
@@ -98,6 +100,8 @@ def run(self):
                         area=event_data["area"],
                         has_clip=event_data["has_clip"],
                         has_snapshot=event_data["has_snapshot"],
+                        sub_label="",
+                        retain_indefinitely=False,
```

Comment:
You don't need to specify these values if they are the defaults for the model. Also, this would clear the flag for users that starred the event before it ended.